### ISSUE TITLE
Support for go module + vendor[🙏]

### ISSUE THREAD

**Issue by cstevandy** [snyk/snyk#1122]
- `Language / Project Type:` golang
- `Package Manager / Build Tool:` go mod vendor

**Describe the user need**
When I ran snyk test/monitor snyk tried to download all dependencies in go mod vendor mode. This causes an error "go: error loading module requirements" in a restricted network or dependency in private repo. I want Snyk to scan a golang project that uses go modules + vendor without trying to download dependencies.

**Describe expected behaviour**
Snyk scans directly without trying to download go dependencies (all dependencies are already available in the vendor folder)

**Additional context**
-

**Comment by lili2311** [comment]
Please refer to the answer here: https://github.com/snyk/snyk/issues/1127#issuecomment-646205732 Does this clarify?

**Comment by cstevandy** [comment]
Yes if it's only the go.mod, but in my case all dependencies/packages are already in the vendor folder. All required information is already provided in the folder.

**Comment by orsagie** [comment]
Hey @cstevandy we have not yet looked into supporting go.mod + vendor. Tagging this as a feature request.

**Comment by RalphBragg** [comment]
Plus one for this feature
